scripts/functional_tests.js

```diff
@@ -20,7 +20,6 @@ const onlyNotInCoverageTests = [
   require.resolve('../test/api_integration/config.js'),
   require.resolve('../test/interpreter_functional/config.ts'),
   require.resolve('../test/examples/config.js'),
-  require.resolve('../test/functional_execution_context/config.ts'),
```

Comment:
I don't see the `test/functional_execution_context` folder/files being moved or deleted in the PR, should we delete it?